easybuild/easyconfigs/c/ctffind/ctffind-4.1.13-foss-2019a.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+#
+# Author: Ake Sandgren, HPC2N, Umea University
+
+easyblock = 'ConfigureMake'
+
+name = 'ctffind'
+version = '4.1.13'
+
+homepage = 'http://grigoriefflab.janelia.org/ctffind4'
+description = """Program for finding CTFs of electron micrographs."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://grigoriefflab.janelia.org/sites/default/files/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['ctffind-4.1.13_remove_bogus_include_lines.patch']
+checksums = [
+    '48231f8511b222176ea39b80c58ae8fb8a6bac5e0da247c54f5a84b52c8750cf',  # ctffind-4.1.13.tar.gz
+    # ctffind-4.1.13_remove_bogus_include_lines.patch
+    'ae218a61a24cec2e35fa4a7ddd497c0d1bb997957bfeb8866d941442afe26365',
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.2'),
+    ('LibTIFF', '4.0.10'),
+    ('GSL', '2.5'),
+    ('wxWidgets', '3.0.4'),
+]
+
+separate_build_dir = True
```

Comment:
@akesandgren This is meaningless when using `ConfigureMake`, only makes sense with `CMakeMake`?

Reply:
Yeah, but a correctly written autoconf setup can also use separate build dirs :-)

Dropping...